policyengine_us/parameters/gov/states/mi/tax/income/exemptions/dependent_on_other_return.yaml

```diff
@@ -0,0 +1,27 @@
+description: Michigan provides this exemption amount for a taxpayer who can be claimed as a dependent on someone else’s return. 
+SINGLE:
+  2021-01-01: 1_500
+  2022-01-01: 0
```

Comment:
What's the source for this going away in 2022? I still see it in subsection 4 of [the law](https://www.legislature.mi.gov/(S(ialnnrioc3xaxwpvjcjssnos))/mileg.aspx?page=getobject&objectname=mcl-206-30#:~:text=206.30%20%22Taxable%20income%22%20defined%3B,432.47%20from%20adjusted%20gross%20income)

Reply:
this exemption didn't exist in [Michigan Taxpayer's Guide 2022](https://www.legislature.mi.gov/Publications/TaxpayerGuide.pdf).

Reply:
This exemption is also not mentioned on the [gov. website](https://www.michigan.gov/taxes/iit/new-developments-for-tax-year-2022) for 2022.

Reply:
It does

>9. EXEMPTIONS. NOTE: If someone else can claim you as a dependent, check box 9e, enter 0 on line 9a and enter $1,500 on line 9e (see instr.).

https://www.legislature.mi.gov/Publications/TaxpayerGuide.pdf#page=23

Reply:
u r right, I'll change 2022 value into 1500